### DISCUSSION TITLE
Low Lighthouse Performance Score - Entry Too Big

### DISCUSSION THREAD

**Discussion by arashctl** [nuxt/nuxt#16893] [Questions] [answered]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0-rc.8`
- Package Manager:  `yarn@1.22.17`
- Builder:          `vite`
- User Config:      `privateRuntimeConfig`, `publicRuntimeConfig`, `buildModules`, `css`, `build`
- Runtime Modules:  `-`
- Build Modules:    `@nuxtjs/tailwindcss@5.3.2`, `@pinia/nuxt@0.4.1`, `@vueuse/nuxt@8.9.4`
------------------------------

### Reproduction

Here it is, but I couldn't get a lighthouse score for the output. 

https://stackblitz.com/edit/nuxt-starter-qeunif?file=package.json,nuxt.config.ts

### Describe the bug

I'm getting a very low performance score from Lighthouse (33/100). You can see my packages and config in the reproduction. 

The entry file is too big, and I see a lot of css in it from these packages: 
* vue final modal
* google recaptcha
* huge chunk of file is related to fontawesome (I have the pro account, and I've added quite a few icons to its library through a plugin)
* Primevue

![Untitled](https://user-images.githubusercontent.com/38472389/187294396-24098f5a-de67-401d-88f3-7a0cceff689c.png)

**Comment by danielroe** [comment, marked as answer]
The issue isn't reproducible with your reproduction. (The entry is 96kb.) It is almost certainly that you are importing vast amounts of JS into your entry, likely within a plugin. I would recommend using the tree-shaking capabilities of the libraries, utilising dynamic imports where possible, etc.

**Comment by kstraszewski** [comment]
Moment.js is a perfo killer. So Moment.js -> day.js for sure. Also i do not know prime and headlessui, but they should use some kind of loader to tree-shake.
